Hanjo

Bhanjo opened the pull request.

Bhanjo committed:
검색결과 페이지 컨텐츠 추가

Bhanjo committed:
검색결과 페이지 버튼 이벤트 구현

Bhanjo committed:
이미지 alt 수정, 추가

ndaemy commented on actProjects/App/static/css/style.css:
```diff
@@ -3,10 +3,13 @@ html, body {
 	padding: 0;
 	border: 0;
```
요 파일은 공용으로 사용할 부분과 특정 페이지에 종속되는 부분을 쪼개야 할 것 같습니다!

ndaemy commented on actProjects/App/static/css/style.css:
```diff
@@ -97,10 +100,52 @@ hr {
 }
 input[type="radio"][name="filtering"]:checked + .filterLabel {
```
css에서 이런 식으로 다양한 선택자를 한 번에 사용하는 것은 가독성을 떨어트리는 주요 요소가 됩니다
좀 더 specific한 class나 id를 사용하시는걸 추천드려요!

ndaemy commented on actProjects/App/static/css/style.css:
```diff
@@ -97,10 +100,52 @@ hr {
     background-color: rgb(255, 255, 255);
     font-weight: bold;
+}
+.actInner, .acterInner {
```
동일한 두 개의 선택자를 , 로 두 번 사용할 필요는 없어보입니다!

ndaemy commented on actProjects/App/static/css/style.css:
```diff
@@ -97,10 +100,52 @@ hr {
+    justify-content: center;
+    display: flex;
+    flex-wrap: wrap;
+    position: relative;
```
`position: relative`는 top, right, bottom, left 와 같이 사용하시지 않으면 아무 효과가 없습니다
요거는 불필요한 코드입니다

ndaemy commented on actProjects/App/static/js/searchPage.js:
```diff
@@ -0,0 +1,42 @@
+
+const view = [];
+for(let i = 0; i < 3; i++) {
+    view[i] = document.getElementById('view' + i);
```
```view.push(document.getElementById('view' + i);```
요런식으로도 가능합니다!

ndaemy commented on actProjects/App/templates/searchPage.html:
```diff
@@ -16,53 +14,121 @@
+            -->
+            <h1 align="center">'공연'의 검색결과입니다</h1>
+            <div>
+                <div class="contentTitle" style="justify-content: start;"><h2>공연</h2>&nbsp;<a onclick="changeAct()">더보기</a></div>
```
inline style은 추천하지 않아요. 차라리 contentTitle에 `justify-content:start;` 를 넣는 것을 추천드립니다

ndaemy reviewed (changes requested):
js 공부 좀 하셨는데요? ㅋㅋㅋㅋ

Bhanjo committed:
코드리뷰 피드백 반영

css와 js 피드백을 반영했습니다.

Bhanjo committed:
태그 내 align 속성 제거

Bhanjo commented:
`search-play-with-options-style.css`
`header-style.css`
혹시 vw, vh 단위들을 다른 단위로 바꿔주실 수 있나요?
브라우저창 확대 비율에 따라 스타일이 다 망가집니다ㅠ

Bhanjo committed:
css파일 분리, 극단 이미지 추가 및 버튼 디자인 수정

css 파일 중복을 피하고자 파일을 분리했습니다.
극단 이미지를 추가해 심미성을 향상시켰습니다.
공연 버튼 디자인을 수정해 마우스 hover시 클릭을 할 수 있음을
한번에 알 수 있게 했습니다

Bhanjo committed:
수정 : css 정렬 기준 수정, 쉐도우 개선

Bhanjo committed:
추가 : 라디오 버튼 동작 추가

프로토타입 퀄리티 향상을 위해 라디오 버튼을 클릭했을 때 추가적인 컨텐츠가 나오도록 기능을 추가했습니다

Bhanjo committed:
추가 : 극단 개별페이지 추가

Bhanjo committed:
추가 : 극단 구성원 슬라이드 구현

한 페이지에서 모든 극단 구성원을 보여주기 위해 슬라이드 기능을 추가했습니다

Bhanjo committed:
수정 : 일부 css 스타일 수정

Bhanjo committed:
스타일 : 단어 오타 수정

Bhanjo committed:
수정 : 극단 구성원 프로필 수정

Bhanjo committed:
추가 : 배너 슬라이드 구현

Bhanjo committed:
추가 : 극단 구성원 스크롤 애니메이션 추가

ndaemy commented on actProjects/App/static/js/searchPage.js:
```diff
@@ -10,11 +10,16 @@ for(let i = 0; i < 3; i++) {
-    title[0].style.display = title[1].style.display = title[2].style.display = 'flex';
-    hr[1].style.display = hr[2].style.display='block';
+    // 임시로 전체 버튼을 클릭하면 새로 추가된 요소 삭제를 위함
+    window.location.href = 'http://127.0.0.1:8000/searchPage';
```
여기는 `window.location.href = '/searchPage'` 라고 하시던지, `window.location.reload()` 라고 하시는걸 추천드려요.
나중에 서버를 배포한다고 가정했을 때, 그 서버는 127.0.0.1:8000 이 아닐 것이기 때문에, 이렇게 되면 없는 페이지로 가는 상황이 생깁니다.

ndaemy commented on actProjects/App/static/css/theater_detail.css:
```diff
@@ -0,0 +1,107 @@
+}
+.acter_wrap {
+    margin: 50px 0px;
+}
```
위에 것 처럼, 값이 0 일 때는 단위를 제거해주시는 걸 추천드립니다!

ndaemy commented on actProjects/App/static/css/theater_detail.css:
```diff
@@ -0,0 +1,107 @@
+}
+.act_wrap {
+
+}
```
css 적용하실 것이 없다면, 선택자를 굳이 넣지 않아도 돼요!

ndaemy commented on actProjects/App/templates/theater_detail.html:
```diff
@@ -46,82 +47,194 @@ <h2>공지</h2>
+                        </a>
+                        <p>마지막</p>
+                        <p>직책</p>
+                    </div>
```
와 이거 제발 진짜 DB로 넣어서 이런 개고생 안하기를..

Bhanjo replied:
ㅋㅋㅋㅋㅋㅋㅋ큐ㅠㅠㅠㅠㅠ

ndaemy commented on actProjects/App/static/css/theater_detail.css:
```diff
@@ -44,16 +43,24 @@
     margin: 10px 0;
 }
+.notion_post h2::before, .notion_post h2::before {
+    content: '';
```
음..? 요건 뭐죠?

ndaemy commented on actProjects/App/static/js/searchPage.js:
```diff
@@ -100,7 +100,7 @@ function addMoreActor() {
         const imgEle = document.createElement('img');
         imgEle.className = 'actorImg';
-        imgEle.src = 'http://127.0.0.1:8000/static/img/actor.png';
+        imgEle.src = 'http://127.0.0.1:8000/static/img/actor' + (i + 6) + '.jpg';
```
요거를 쉽게 해주는 문법 중에 JS [Template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) 이라는 녀석이 있습니다.
여기서는
```javascript
  imgEle.src = `http://127.0.0.1:8000/static/img/actor${i + 6}.jpg`;
```
라고 쓸 수 있어요.

ndaemy commented on actProjects/App/static/css/theater_detail.css:
```diff
@@ -6,39 +6,48 @@
+.slide_btn_prev, .slide_btn_next {
+    position: absolute;
+    cursor: pointer;
+    top: 38%;
```
이런거 % 단위면 나중에 깨질 수도 있습니다! top 같은 속성은 고정 단위 (ex. px) 를 사용하시면 좋아요

Bhanjo committed:
수정 : 코드리뷰 피드백 반영
